charts/orchestrator/values.yaml

```diff
@@ -115,6 +115,10 @@ orchestrator:
       limits:
         memory: "1Gi"
         cpu: "500m"
+    dataIndex:
+      image: "quay.io/kiegroup/kogito-data-index-postgresql-nightly:latest" # To be removed when stable version is released
+    jobService:
+      image: "quay.io/kiegroup/kogito-jobs-service-postgresql-nightly:latest" # To be removed when stable version is released
```

Comment:
I don't think we need these comments (or they need to be updated as the value will never be "Removed: as they say)

Reply:
is this form better?/
```
#TODO  change to stable image when stable version is out"
image: ...
```